### ISSUE TITLE
Features on View Type (Download and alignment )

### ISSUE THREAD

**Issue by marciox25** [laurencedawson/sync-for-reddit#945]
**Is your feature request related to a problem? Please describe.**
If I remember correctly, before adquiring the PRO version, we had this option. Where an download button would be near the "save" (yellow flag on image) option. Altough this option is still present, we would need to click the image before we can see the download button. It would be really convenient to simply have the download button near the save/share/mod options as an (on/off) feature too.

What I miss is an "costumizable" alignment for the buttons bellow. What if we only want to see Upvote/save and download? Or even change them from places from right to left or vice-versa. 

These addons would be a really nice feature for sure!

**Describe the solution you'd like**
As mentioned above, it would nice to have at least the instant download button available. The Further aligning and choosing position of the elements would be a nice to have too.

**Describe alternatives you've considered**
There are no other alternatives for this issue. Downloading would require the extra click on the image, and for right aligment is not really what I expected to be.

**Additional context**
![image](https://user-images.githubusercontent.com/14317256/160585017-09795dfa-d2dc-493a-85c2-b10c6e029e2e.png)
